### PR TITLE
fix: Search dropdown redirect

### LINKED ISSUE #5258: Search dropdown redirect
image
wrong redirect 

search something on landing page 
image
- then switch tab to NFTs - it redirects you to the explore, but it should reveal the items tab instead

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5258
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/search/Search.vue [hunk 1]
```diff
@@ -210,6 +210,10 @@ export default class Search extends mixins(
     return this.preferencesStore.getReplaceBuyNowWithYolo
   }
 
+  get isSearchPage() {
+    return this.routePathList.indexOf(this.$route.path) !== -1
```
Comment (daiagi) [inline components/search/Search.vue:214]:
```suggestion
    return this.routePathList.includes(this.$route.path)
```
easier to read no?

also, maybe calling the function ```isExplorePage``` is a more fitting name?

**Review by daiagi — CHANGES_REQUESTED** [review]

File: components/search/Search.vue (hunk 1 above)
Comment (vikiival) [reply components/search/Search.vue]: Agree 
a70256b

**Comment by vikiival** [comment]
https://stackoverflow.com/questions/35370222/array-prototype-includes-vs-array-prototype-indexof

File: components/search/Search.vue (hunk 1 above)
Comment (Jarsen136) [reply components/search/Search.vue]: > also, maybe calling the function isExplorePage is a more fitting name?

Yes, updated it

**Comment by vikiival** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 33.76 USD/KSM ~ 0.592 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xf88881bc296b36308ce40cafed8b959330c3a4b6b07963a3e4102dd9c9e0002e](https://kusama.subscan.io/extrinsic/0xf88881bc296b36308ce40cafed8b959330c3a4b6b07963a3e4102dd9c9e0002e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**